sources/Filer/HelpTab.cpp

```diff
@@ -178,7 +178,10 @@ HelpTab::HelpTab()
 				.Add(fDocsButton)
 				.AddGlue()
 			.End()
+		.AddGlue(100.0)
 		.End();
+
+	SetExplicitMaxSize(BSize(B_SIZE_UNLIMITED, B_SIZE_UNLIMITED));
```

Comment:
Try AddStrut() instead of AddGlue() and then remove this. It *should* work but it probably won't, because Haiku's layout system is funky...